adagrams/game.py

```diff
@@ -1,11 +1,148 @@
+import random
+
+def create_letter_pool():
+
+    letter_pool = []
+
+    letter_frequencies = {
```

Comment:
Nice choice placing the letter frequencies in the function to keep the data close to where it's used & ensure multiple calls won't change the data.